seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -451,9 +451,9 @@
         "</div>"
       ],
       "tests": [
-        "assert($('u').length == 1, 'message: Your code should add a <code>u</code> tag to the markup.');",
-        "assert($('u').children('a').length == 2, 'message: The <code>u</code> tag should wrap around and contain the two <code>a</code> tags.');",
-        "assert($('u').children('div').length == 0, 'message: The <code>u</code> tag should not wrap around the parent <code>div</code> tag.');"
+        "assert($('u').length === 1, 'message: Your code should add a <code>u</code> tag to the markup.');",
+        "assert($('u').text().indexOf('Ph.D. students') > -1, 'message: The <code>u</code> tag should wrap around the text \"Ph.D students\".');",
```

Comment:
@Greenheart no prob! Yeah, it blew my mind when I found out you can do that. The "language" for the code block is "diff" and you just have to add the `+` and `-` signs accordingly.